### ISSUE TITLE
Migrate to WalletConnect 

### ISSUE THREAD

**Issue by d-mooers** [mobiusAMM/mobius-interface#169]
**Is your feature request related to a problem? Please describe.**
UseContractKit is kind of a mess.  The Valora integration works maybe 50% of the time, and routinely opens to the incorrect page.  WalletConnect and other Web3 libraries are much better maintained, and will always to go multichain in the future (if desired, this does not mean we ARE going multichain).

**Describe the solution you'd like**
Migrate our web3 libraries from useContractKit to Web3-React-Modal.

**Additional context**
This will require us to re-do our wallet connect flow.

**Comment by kyscott18** [comment]
Would this effectively deprecate valora?

**Comment by d-mooers** [comment]
> Would this effectively deprecate valora?

No, but I think we can just upgrade to uck3.0 instead of switching to WalletConnect for now

**Comment by kyscott18** [comment]
Tried to update to uck 3.0.0 yesterday and valora integrations were failing
